Xamarin.Forms.Core/InputView.cs

```diff
@@ -1,3 +1,9 @@
+using Xamarin.Forms;
+using Xamarin.Forms.StyleSheets;
+
+[assembly: StyleProperty("-xf-inputview-max-length", typeof(InputView), nameof(InputView.MaxLengthProperty))]
```

Comment:
CSS doesn't expose `max-length` for a reason. we shouldn't either

Reply:
No problemo. I only added it as it was in the spec. Did I read the spec wrong or is the spec wrong?

Reply:
> Spec is wrong

I drafted the spec, and I agree

Reply:
`-xf-max-length`